api-report/driver-utils.api.md

```diff
@@ -365,8 +373,8 @@ export function streamFromMessages(messagesArg: Promise<ISequencedDocumentMessag
 export function streamObserver<T>(stream: IStream<T>, handler: (value: IStreamResult<T>) => void): IStream<T>;
 
 // @public
-export class ThrottlingError extends LoggingError implements IThrottlingWarning, IFluidErrorBase {
-    constructor(fluidErrorCode: string, message: string, retryAfterSeconds: number, props?: ITelemetryProperties);
+export class ThrottlingError extends DriverErrorWithVersion implements IThrottlingWarning, IFluidErrorBase {
+    constructor(fluidErrorCode: string, message: string, retryAfterSeconds: number, driverVersion: string, props?: ITelemetryProperties);
```

Comment:
why can't we just reuse the props and add those to exceptions?

Reply:
I want each type's constructor to require the driver version, rather than counting on callers to remember to add it to props (this is where I started but then realized it was impossible to enforce)